src/components/navigation/_macro.njk

```diff
@@ -68,7 +68,7 @@
                 "type": "button",
                 "variants": ["mobile", "dropdown"],
                 "attributes": {
-                    "aria-label": "Toggle section navigation",
+                    "aria-label": "Toggle " + params.navigation.currentPageTitle + " menu" if params.navigation.currentPageTitle else "Toggle section menu",
```

Comment:
Seems like the syntax here is wrong. Doesn't the `if` need to go first? If `params.navigation.currentPageTitle` doesn't exist then it will fall over I think.

Reply:
It does look a bit weird I agree, seems to be similar syntax used in other places when passing props. Also tested it and it works. SS below is from `example-header-external-with-sub-navigation.njk` with currentPageTitle removed.

![Screenshot 2023-07-27 at 11 16 07](https://github.com/ONSdigital/design-system/assets/131372109/1d40b3d6-afc6-4fbe-b971-3736ac8070b2)